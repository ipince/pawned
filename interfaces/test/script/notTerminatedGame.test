StartNewGame human 0 human 0
MakeMove b2-b3 0
MakeMove b7-b5 0
MakeMove a2-a4 0
MakeMove b5-a4 0
MakeMove b3-a4 0
MakeMove g7-g6 0
MakeMove g2-g3 0
MakeMove f8-h6 0
MakeMove f2-f3 0
MakeMove h6-d2 0
MakeMove d1-d2 0
MakeMove g8-h6 0
MakeMove d2-d7 0
MakeMove c8-d7 0
MakeMove c1-h6 0
MakeMove d7-a4 0
MakeMove a1-a4 0
MakeMove b8-a6 0
MakeMove a4-a6 0
MakeMove g6-g5 0
MakeMove a6-a7 0
MakeMove a8-a7 0
MakeMove h6-g5 0
MakeMove d8-d2 0
MakeMove e1-d2 0
#QuitGame