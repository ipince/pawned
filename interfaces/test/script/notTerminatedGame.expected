New game started
b2-b3
b7-b5
a2-a4
b5-a4
b3-a4
g7-g6
g2-g3
f8-h6
f2-f3
h6-d2
d1-d2
g8-h6
d2-d7
c8-d7
c1-h6
d7-a4
a1-a4
b8-a6
a4-a6
g6-g5
a6-a7
a8-a7
h6-g5
d8-d2
e1-d2
#QuitGame
